spec/models/course/condition/assessment_spec.rb

```diff
@@ -111,62 +111,86 @@
       end
 
       context 'when there is minimum grade percentage' do
-        subject do
-          condition = create(:course_condition_assessment, minimum_grade_percentage: 60)
-          condition.assessment = assessment
-          condition
-        end
-        # This submission is graded but its grade is below the minimum grade percentage to satisfy
-        # the condition.
-        let(:below_minimum_grade_submission) do
-          submission = create(:submission, workflow_state: :graded, assessment: assessment,
-                                           user: course_user.user)
-          answers = assessment.questions.attempt(submission)
-          answers.each do |answer|
-            answer.finalise!
-            answer.grade = 0
-            answer.save!
-          end
-          submission
-        end
-        let(:submission) do
-          create(:submission, workflow_state: :graded, assessment: assessment,
-                              user: course_user.user)
+        let(:condition) do
```

Comment:
Because of the nil maximum grade test case, the subject becomes the condition that contains the proper assessment.